encoders/nlp/SpacyTextEncoder/__init__.py

```diff
@@ -0,0 +1,101 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+
+class SpacyTextEncoder(BaseTorchEncoder):
+    """
+    :class:`SpacyTextEncoder` encodes data from a `np.ndarray` (of strings) of length `BatchSize` into
+    a `np.ndarray` of shape `Batchsize x EmbeddingDimension`.
+
+    :param lang: pre-trained spaCy language pipeline (model name HashEmbedCNN by default for tok2vec), `en_core_web_sm` by default.
```

Comment:
Maybe we can add the possible values for 'lang'?

Reply:
Sure, done.